test/test.js

```diff
@@ -2301,6 +2301,18 @@ describe('Page', function() {
       await page.keyboard.type('Hello World!');
       expect(await page.evaluate(() => textarea.value)).toBe('He Wrd!');
     });
+    it('should remove keys from _pressedKeys after keyboard.up()', async({page, server}) => {
+      await page.goto(server.PREFIX + '/input/textarea.html');
+      await page.focus('textarea');
+      await page.keyboard.down('w');
+      expect(await page.keyboard._pressedKeys.size).toEqual(1);
```

Comment:
@travch It's really unfortunate to access private field here. Can we come up with a test that uses public API only?